sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/ConfigPropertiesTest.java

```diff
@@ -216,4 +220,31 @@ void durationUnitParsing() {
                 .getDuration("duration"))
         .isEqualTo(Duration.ofMillis(8));
   }
+
+  @Test
+  void defaultMethods() {
+    ConfigProperties properties = DefaultConfigProperties.get(emptyMap());
+    assertEquals(true, properties.getBoolean("foo", true));
```

Comment:
Use assertj for all the assertions.

Reply:
Yes, we want to use assertJ for everything. If you know of tests that aren't, please create an issue pointing to them.